spec/services/symphony_client_spec.rb

```diff
@@ -145,4 +145,135 @@
       expect(bib_response.body.to_str).to include('/catalog/bib')
     end
   end
+
+  describe '#place_hold' do
+    before do
+      stub_request(:post, uri)
```

Comment:
I see because you did the stub_requests here.

Reply:
yes, that is why, and also my thinking was that the controller specs should not know about what the ws call is, so I did not use webmock approach in the controller but used it here for the client spec.